CONTRIBUTING.md

```diff
@@ -289,7 +289,8 @@ For convenience you can filter the regeneration by target (api, lua, lsp) using
 
 ### Lua docstrings
 
-Lua documentation uses a subset of [EmmyLua] annotations. See [:help dev-doc-lua][dev-doc-lua].
+Use [LuaLS] annotations in Lua docstrings to annotate parameter types, return
+types, etc. See [:help dev-doc-lua][dev-doc-lua].
```

Comment:
we're using luals annotations, not emmylua, right? @clason

Reply:
Well, emmylua isn't really active anymore so I think LuaLS is redefining it.

See: https://github.com/LuaLS/lua-language-server/issues/980#issuecomment-1082841463